flink-libraries/flink-table/src/main/scala/org/apache/flink/table/catalog/ExternalCatalogTable.scala

```diff
@@ -37,25 +36,10 @@ import org.apache.flink.table.plan.stats.TableStats
   * @param lastAccessTime       Timestamp of last access of the table
   */
 case class ExternalCatalogTable(
-    identifier: TableIdentifier,
```

Comment:
I agree that it requires keeping more than the name of the table. The more general problem is that you will need to keep additional information for other data sources (e.g., the clusters for Kafka topics)

I think it might make more sense to put the information at the properties field in `ExternalCatalogTable`. Thoughts?